dist/api_data.py

```diff
@@ -1339,7 +1341,7 @@ def __cmp__(self, other):
         \c oldest_timestamp and the read timestamps of all active readers, and
         \c stable returns the most recent \c stable_timestamp set with
         WT_CONNECTION::set_timestamp.  See @ref transaction_timestamps''',
-        choices=['all_committed','oldest','pinned','stable']),
+        choices=['all_committed','oldest','pinned','recovery','stable']),
```

Comment:
Is the `recovery` timestamp actually the `checkpoint` timestamp? Could we call it that, and have it more generally useful?

Reply:
MongoDB only needs to query the value that existed on startup. For an online rollback calling `rollback_to_stable`, MongoDB knows precisely what it has set the stable timestamp to.

This certainly doesn't preclude that there may be some other use case. Perhaps having `recovery` now that is guaranteed to remain constant for the lifetime of the program, and, if it becomes useful, later adding in a `checkpoint` value later that tracks checkpoints as they happen?